vivisect/parsers/elf.py

```diff
@@ -3,20 +3,26 @@
 import logging
 
 import Elf
-import envi
 import vivisect
 import vivisect.parsers as v_parsers
 import envi.bits as e_bits
 
 from vivisect.const import *
 
-from cStringIO import StringIO
+from io import StringIO
+
 
 logger = logging.getLogger(__name__)
 
+# FIXME: So this is terrible, but until we unite everything under one package
+# namespace to rule them all, we need this to avoid a stupid amount of logspam
+# but we can't just force things under
+elog = logging.getLogger(Elf.__name__)
```

Comment:
what are we trying to do here?

Reply:
It forces the logger for the Elf module to follow whatever the logging level for vivisect/parsers/elf.py is.